src/main/java/com/example/hbs/model/Booking.java

```diff
@@ -1,32 +1,45 @@
 package com.example.hbs.model;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import javax.persistence.*;
 import java.time.LocalDate;
 
 @Entity
-@Table
 @Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "bookings")
 public class Booking {
     @ManyToOne
-    @JoinColumn
+    @JoinColumn(nullable = false)
```

Comment:
if a user must be present for a given Booking, then the booking should also be deleted when then the user is deleted.